src/Scroll.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import { BrowserRouter as Router, withRouter } from 'react-router-dom'
+
+class ScrollToTop extends React.Component {
```

Comment:
Ideally filename matches classname; not strictly necessary, but adds consistency.